starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,40 @@
-// Names and Input
+var hacker1;
+hacker1 = "claudia";
+console.log("the driver's name is" + " " + hacker1 );
+var hacker2;
+hacker2 = "summer";
+console.log("the navigator's name is" + " " + hacker2);
+hacker1.length = 7;
+hacker2.length = 6;
+if (hacker1< hacker2) {
+  console.log("The driver's name is longer")
+}
+else if (hacker2 > hacker1) {
+  console.log("The navigaor has the longest name")
+}
+else if (hacker1 === hacker2) {
+  console.log("both names are long");
+}
+
```

Comment:
to find out which var is longer, you just need to compare hacker1.length to hacker2.length .  there's no need to hacker.length =..